src/avram/migrator/runner.cr

```diff
@@ -137,6 +137,12 @@ class Avram::Migrator::Runner
     end
   end
 
+  def setup_migration_tracking_tables
```

Comment:
Could you make this a class method instead? I think that might make more sense since it doesn't need an instance variables

Reply:
Updated in https://github.com/luckyframework/avram/pull/135/commits/f5dd3b12bd18c229595a83777f649b5757997636

Note that I also pulled up `create_table_for_tracking_migrations` into a private class method, since it too does not depend on instance variables but is called by `setup_migration_tracking_tables`